src/main/java/org/opentripplanner/hasura_client/HasuraGetter.java

```diff
@@ -25,10 +30,16 @@ protected boolean addGeolocationArguments() {
     }
 
     private String getGeolocationArguments(Graph graph) {
-        double latMin = graph.getEnvelope().getLowerLeftLatitude();
-        double lonMin = graph.getEnvelope().getLowerLeftLongitude();
-        double latMax = graph.getEnvelope().getUpperRightLatitude();
-        double lonMax = graph.getEnvelope().getUpperRightLongitude();
+        List<OsmVertex> vertices = graph.getVertices().stream()
```

Comment:
a nie lepiej to zrobić w `Graph` pole ` WorldEnvelope osmVerticesEnvelope` wyliczone raz? Analogicznie do tego:
```
    //Envelope of all OSM and transit vertices. Calculated during build time
    private WorldEnvelope envelope = null;
...
    /**
     * Calculates envelope out of all OSM coordinates
     * <p>
     * Transit stops are added to the envelope as they are added to the graph
     */
    public void calculateEnvelope() {
        this.envelope = new WorldEnvelope();

        for (Vertex v : this.getVertices()) {
            Coordinate c = v.getCoordinate();
            this.envelope.expandToInclude(c);
        }
    }
```

Reply:
Racja, zmieniłem jak zaproponowałeś